mavros_extras/src/plugins/debug.cpp

```diff
@@ -0,0 +1,254 @@
+/**
+ * @brief Debug messages plugin
+ * @file debug.cpp
+ * @author Nuno Marques <n.marques21@hotmail.com>
+ *
+ * @addtogroup plugin
+ * @{
+ */
+/*
+ * Copyright 2017 Nuno Marques.
+ *
+ * This file is part of the mavros package and subject to the license terms
+ * in the top-level LICENSE file of the mavros repository.
+ * https://github.com/mavlink/mavros/tree/master/LICENSE.md
+ */
+
+#include <mavros/mavros_plugin.h>
+
+#include <mavros_msgs/Debug.h>
+
+namespace mavros {
+namespace extra_plugins {
+//! @brief Plugin for Debug msgs from MAVLink API
+class DebugPlugin : public plugin::PluginBase {
```

Comment:
Please name it `debug_value` (and `debug_value.cpp`).

Reply:
sure